src/e2e/tatum.rates.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiVersion, Network, TatumSDK } from "../service";
+import { Ethereum } from "../dto";
+
+describe("Rates", () => {
+  let tatum: TatumSDK<Ethereum>
+  beforeAll(async () => {
+    tatum = await TatumSDK.init<Ethereum>({
+      network: Network.ETHEREUM_SEPOLIA,
+      verbose: true,
+      retryDelay: 1000,
+      retryCount: 2,
+      version: ApiVersion.V2,
+    })
+  })
+  it("get ETH/EUR", async () => {
+    const res = await tatum.rates.getCurrentRate("BTC","EUR");
+    console.log(res)
```

Comment:
Please remove console logs, when your tests are stable, we need to clear the logs.

Reply:
removed